opencti-platform/opencti-graphql/tests/04-sync/00-Raw/sync-raw-test.js

```diff
@@ -40,7 +40,7 @@ describe('Database sync raw', () => {
   it(
     'Should python raw sync succeed',
     async () => {
-      const queryResultBefore = await queryAsAdmin({ query: LIST_QUERY, variables: { first: 350 } });
+      const queryResultBefore = await queryAsAdmin({ query: LIST_QUERY, variables: { first: 500 } });
```

Comment:
because I got an error in count here, I think we have more test so we need to get a bit more data.